### PR TITLE
Update committer list

### PR DESCRIPTION
**Pull request by hcho3**
Update the committer list in `CONTRIBUTORS.md` to include everyone who has write access to this repository.

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -6,21 +6,30 @@ Committers
...
-  - Tong is an applied scientist in Amazon AI, he is the maintainer of xgboost R package.
+  - Tong is an applied scientist in Amazon AI. He is the maintainer of XGBoost R package.
 * [Vadim Khotilovich](https://github.com/khotilov)
   - Vadim contributes many improvements in R and core packages.
 * [Bing Xu](https://github.com/antinucleon)
-  - Bing is the original creator of xgboost python package and currently the maintainer of [XGBoost.jl](https://github.com/antinucleon/XGBoost.jl).
+  - Bing is the original creator of XGBoost Python package and currently the maintainer of [XGBoost.jl](https://github.com/antinucleon/XGBoost.jl).
 * [Michael Benesty](https://github.com/pommedeterresautee)
-  - Micheal is a lawyer, data scientist in France, he is the creator of xgboost interactive analysis module in R.
+  - Micheal is a lawyer and data scientist in France. He is the creator of XGBoost interactive analysis module in R.
```
Comment (terrytangyuan) [inline CONTRIBUTORS.md]: Misspell: Micheal -> Michael

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -6,21 +6,30 @@ Committers
...
 * [Yuan Tang](https://github.com/terrytangyuan)
   - Yuan is a data scientist in Chicago, US. He contributed mostly in R and Python packages.
```
Comment (terrytangyuan) [inline CONTRIBUTORS.md]: Could you update my information to: " a software engineer in Ant Financial"? Thanks.

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -6,21 +6,30 @@ Committers
...
-* [Nan Zhu](https://github.com/CodingCat)
-  - Nan is a software engineer in Microsoft. He contributed mostly in JVM packages.
-* [Sergei Lebedev](https://github.com/superbobry)
-  - Serget is a software engineer in Criteo. He contributed mostly in JVM packages.
+* [Nan Zhu](https://github.com/CodingCat), Uber
+  - Nan is a software engineer in Uber. He contributed mostly in JVM packages.
+* [Sergei Lebedev](https://github.com/superbobry), Criteo
+  - Sergei is a software engineer in Criteo. He contributed mostly in JVM packages.
+* [Hongliang Liu](https://github.com/phunterlau)
```
Comment (terrytangyuan) [inline CONTRIBUTORS.md]: cc @phunterlau Wanna add some description here?

**Comment by terrytangyuan** [comment]
@hcho3 I just realized that I can directly push to your branch so I just made the changes.

**Comment by hcho3** [comment]
@terrytangyuan Thanks!

**Pull request merged**